src/android/ForeSeeAPI.java

```diff
@@ -757,13 +760,13 @@ public boolean invoke(final JSONArray args, final CallbackContext callback, Cord
                     mCallbacks.clear();
 
                     //2. 
-                    Predictive.setInviteListener(new FSCordovaInviteListener());
+                    Predictive.setInviteListener(new XMCordovaInviteListener());
```

Comment:
This should probably be as follows to fit in with other SDKs
```suggestion
                    Predictive.setInviteListener(new EXPCordovaInviteListener());
```